tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_container_validation.py

```diff
@@ -348,22 +348,22 @@ def test_valid_containers(
         ),
         Container(
             name="zero_size_container_section",
-            raw_bytes="ef00 01 01 0004 02 0001 0001 03 0001 0000 ff 0000 00 00800000 00",
+            raw_bytes="ef00 01 01 0004 02 0001 0001 03 0001 00000000 ff 0000 00 00800000 00",
```

Comment:
I think you should add 2 more tests as the above: for the incomplete container sections size.